packages/common/src/dual-list-select/dual-list-select.js

```diff
@@ -33,7 +33,9 @@ const DualListSelectCommon = (props) => {
 
   const { DualListSelect, ...rest } = useFieldApi({
     ...props,
-    isEqual: (current, initial) => isEqual([...(current || [])].sort(), [...(initial || [])].sort())
+    FieldProps: {
+      isEqual: (current, initial) => isEqual([...(current || [])].sort(), [...(initial || [])].sort())
```

Comment:
Can you add a test here to see if the default prop is properly used?

Reply:
Oh, this is common package - it isn't tested at all because of the build process, see #https://github.com/data-driven-forms/react-forms/issues/932

However, the components using it are all tested in their mappers.